### PR TITLE
New package: Summay.Sunsama version 2.3.4

### PR DESCRIPTION
**Pull request by SpecterShell**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/161083)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

File: manifests/s/Summay/Sunsama/2.3.4/Summay.Sunsama.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PackageVersion: 2.3.4
+InstallerType: nullsoft # electron-builder
+Scope: user
+InstallerSwitches:
+  Upgrade: --updated
+UpgradeBehavior: install
+ReleaseDate: 2024-06-10
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.todesktop.com/2003096gmmnl0g1/Sunsama%20Setup%202.3.4%20-%20Build%20240610gx2gqgnbj-x64.exe
```
Comment (stephengillie) [inline manifests/s/Summay/Sunsama/2.3.4/Summay.Sunsama.installer.yaml:14]:
```suggestion
  InstallerUrl: https://desktop.sunsama.com/
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @SpecterShell,

The `PackageUrl` leads to a slightly different `InstallerUrl`. Should it be used instead?

**Comment by SpecterShell** [comment]
> Hi @SpecterShell,
> 
> The `PackageUrl` leads to a slightly different `InstallerUrl`. Should it be used instead?

The publisher is using the building and updating service provided by [ToDesktop](https://www.todesktop.com/) for this package, and thus the update source of this package is simply set to the ToDesktop site. The current `InstallerUrl` is obtained from auto-updating.

**Comment by SpecterShell** [comment]
> > Hi @SpecterShell,
> > The `PackageUrl` leads to a slightly different `InstallerUrl`. Should it be used instead?
> 
> The publisher is using the building and updating service provided by [ToDesktop](https://www.todesktop.com/) for this package, and thus the update source of this package is simply set to the ToDesktop site. The current `InstallerUrl` is obtained from auto-updating.

I Just read the ToDesktop docs and found these links:
https://dl.todesktop.com/2003096gmmnl0g1/versions/2.3.4/windows
https://desktop.sunsama.com/versions/2.3.4/windows

**Review by stephengillie — APPROVED** [review]

**Pull request merged**